lms/templates/courseware/courseware.html

```diff
@@ -271,3 +271,21 @@
       % endif
```

Comment:
if we're switching over to the new mfe i'm assuming this code won't be used?

Reply:
Yes I'm finding out more about the timing for the various MFEs that are planned. It sounds like the in-course MFE by the T&L team has started rolling out for a percentage of learners (and the rest will still use the old pages), and Seth & Emma are following up to plan for getting these updates reflected in the new code too.